lib/addons/cloudwatch-adot-addon/index.ts

```diff
@@ -42,10 +53,12 @@ export const enum cloudWatchDeploymentMode {
 /**
  * Defaults options for the add-on
  */
-const defaultProps = {
+const defaultProps: CloudWatchAdotAddOnProps = {
     deploymentMode: cloudWatchDeploymentMode.DEPLOYMENT,
     namespace: 'default',
-    name: 'adot-collector-cloudwatch'
+    name: 'adot-collector-cloudwatch',
+    metricsNameSelectors: ['apiserver_request_.*', 'container_memory_.*', 'container_threads', 'otelcol_process_.*', 'ho11y*'],
```

Comment:
we should drop ho11y from here and check out meaningful default metrics that customers want to see. if apiserver requests is an edge case. 
I would rather prefer metrics such as:

node * (CPU, memory, disk, threads)
pod (CPU, memory, threads)
alb ingress (request count, latency)
nginx (whatever they provide ootb, ideally number of http requests, and latency)

I am sure just based on your experience you can come up with a few really meaningful metrics that customers will need.

Reply:
@shapirov103 I have dropped `ho11y` in the metric selectors of the addon but this can be passed from our pattern or any specific customer metric the customer is looking for. We are already getting useful metrics with all `metricNameSelectors` we have already. We will get more metrics once we have `kube-state metrics` and `node-exporter` addons. Please let me know if yu have any questions.